src/main/frontend/util.cljc

```diff
@@ -1374,6 +1374,14 @@
                    (count val))]
        (.setRangeText input "" current (inc idx)))))
 
+#?(:cljs
+   (defn fix-open-external-with-shift!
+     [^js/MouseEvent e]
+     (when (and (.-shiftKey e) win32? (electron?)
```

Comment:
Here why only Windows have this problem?

Reply:
Just be reproduced on Windows platform.